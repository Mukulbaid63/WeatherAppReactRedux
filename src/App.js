import logo from './logo.svg';
import './App.css';

import WeatherMenu from './components/WeatherMenu';
import {BrowserRouter,Switch,Route, Redirect} from "react-router-dom"
import Login from './components/Login';

function App(props) {

 return(
   <BrowserRouter>
   <Switch>
    <Route path="/weather" exact component={WeatherMenu}/>
    <Route path="/" exact component={Login}/>

   </Switch>
   </BrowserRouter>
 )
}
export default App;
