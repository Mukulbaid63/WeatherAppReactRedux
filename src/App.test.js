import { render, screen } from '@testing-library/react';
import App from './App';

// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });
import React from 'react';
import renderer from 'react-test-renderer';



it('renders correctly when there are no items', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});