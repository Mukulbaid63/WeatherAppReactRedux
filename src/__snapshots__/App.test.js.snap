// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="d-flex flex-column align-items-center justify-content-center"
>
  <div
    className="m-4 p-4"
    style={
      Object {
        "border": "1px solid black",
        "borderRadius": "10px",
      }
    }
  >
    <form
      className="d-flex  flex-column align-items-center justify-content-center"
    >
      <h1>
        PLEASE LOGIN: 
      </h1>
      <input
        className="form-control mb-3"
        onChange={[Function]}
        placeholder="Enter your email address..."
        type="email"
        value=""
      />
      <input
        className="form-control mb-3"
        onChange={[Function]}
        placeholder="Enter your password"
        type="password"
        value=""
      />
      <button
        className="btn btn-primary mb-3"
        onClick={[Function]}
      >
        Submit
      </button>
    </form>
  </div>
  <div
    className="row"
  >
    <div
      className="col-12 offset-sm-3 text-left"
    >
      <div
        className="alert alert-danger"
        style={
          Object {
            "display": "none",
          }
        }
      >
        
      </div>
    </div>
  </div>
  <p>
    TEST USER FOR LOGIN:
    <br />
     Email:
    <b>
      user@test.com
    </b>
    <br />
    Password:
    <b>
      test1234
    </b>
  </p>
</div>
`;
